src/printer.ts

```diff
@@ -415,20 +433,26 @@ export class PugPrinter {
 									`code: \`${code.trim()}\``
 								);
 							} else if (error.includes("Unexpected token '('")) {
-								logger.warn(
-									"[PugPrinter:formatText]: Found unexpected token '('. If you are using Vue, you can ignore this message.",
-									`code: \`${code.trim()}\``
-								);
-							} else if (error.includes('Missing expected )')) {
-								logger.warn(
-									'[PugPrinter:formatText]: Missing expected ). If you are using Vue, you can ignore this message.',
-									`code: \`${code.trim()}\``
-								);
-							} else if (error.includes('Missing expected :')) {
-								logger.warn(
-									'[PugPrinter:formatText]: Missing expected :. If you are using Vue, you can ignore this message.',
-									`code: \`${code.trim()}\``
-								);
+								if (pugFramework !== 'vue') {
```

Comment:
**thought:** Mh :thinking:  
We should really find out how `svelte` behaves here :eyes: (and the below ones)

Cause maybe we need to use `pugFramework === 'angular'` here
But I'm absolutely not sure

Reply:
We'll see if some people run into this problem
If so, they will create an issue